libkbfs/keybase_daemon_rpc.go

```diff
@@ -19,32 +18,12 @@ import (
 // KeybaseDaemonRPC implements the KeybaseService interface using RPC
 // calls.
 type KeybaseDaemonRPC struct {
-	context        Context
-	identifyClient keybase1.IdentifyInterface
-	userClient     keybase1.UserInterface
-	sessionClient  keybase1.SessionInterface
-	favoriteClient keybase1.FavoriteInterface
-	kbfsClient     keybase1.KbfsInterface
-	log            logger.Logger
-
-	config Config
+	*KeybaseServiceBase
```

Comment:
Do you need to embed this as a pointer? Didn't need to for `CryptoClientRPC`, but there might be some go thing preventing this.